File: sprint_1_nonfinals/e.cpp
#include <iostream>
#include <string>

using namespace std;

string solve(string input) {
	// Здесь реализация вашего решения

}

void print(string result) {
	cout << result << endl << result.length() << endl;
}

int main() {
	int n;
	cin >> n;
	string s;
	getline(cin, s);
	getline(cin, s);
	print(solve(s));
}

File: sprint_1_nonfinals/python/e.py
def get_longest_word(text: str) -> str:
    # Здесь реализация вашего решения
    pass

def read_input() -> str:
    _ = input()
    text = input().strip()
    return text

def print_result(result: str) -> None:
    print(result)
    print(len(result))

print_result(get_longest_word(read_input()))
